docs/docs/filtering/_category_.yaml

```diff
@@ -0,0 +1,4 @@
+position: 16
+label: Data Filtering
```

Comment:
It _seems_ like this has no effect, since `sidebars.js` determines the sidebars...? Or what is this about? 🤔